php_phongo.c

```diff
@@ -1356,10 +1352,8 @@ void php_phongo_cursor_to_zval(zval *retval, php_phongo_cursor_t *cursor) /* {{{
 		_ADD_BOOL(zcursor, is_command);
 		_ADD_BOOL(zcursor, sent);
 		_ADD_BOOL(zcursor, done);
-		_ADD_BOOL(zcursor, failed);
 		_ADD_BOOL(zcursor, end_of_event);
 		_ADD_BOOL(zcursor, in_exhaust);
-		_ADD_BOOL(zcursor, redir_primary);
```

Comment:
We don't redirect commands anymore, as per spec.
See https://jira.mongodb.org/browse/CDRIVER-807